yvalenta/chess_board.rb

```diff
@@ -0,0 +1,38 @@
+class Chess_board
+
+  def board(f,c)
+    #f = 7
```

Comment:
If you like you hardcode the values is better to use it in the params definition